src/deps.deno.ts

```diff
@@ -0,0 +1,96 @@
+import { Input, Select } from 'https://deno.land/x/cliffy@v0.23.0/prompt/mod.ts';
+import { PromtOptions } from './types.ts';
+import * as path from 'https://deno.land/std@0.136.0/node/path/mod.ts';
+import chalk from 'https://deno.land/x/chalkin@v0.1.3/mod.ts';
+import { TerminalSpinner } from 'https://deno.land/x/spinners@v1.1.2/mod.ts';
+import * as fs from 'https://deno.land/std@0.136.0/node/fs.ts';
+import * as os from 'https://deno.land/std@0.136.0/node/os.ts';
+import * as process from 'https://deno.land/std@0.136.0/node/process.ts';
+import * as stream from 'https://deno.land/std@0.136.0/node/stream.ts';
+import * as util from 'https://deno.land/std@0.136.0/node/util.ts';
+import { Untar } from 'https://deno.land/std@0.136.0/archive/tar.ts';
+import { readerFromStreamReader } from 'https://deno.land/std@0.136.0/streams/conversion.ts';
+import { ensureFile } from 'https://deno.land/std@0.136.0/fs/ensure_file.ts';
+import { ensureDir } from 'https://deno.land/std@0.136.0/fs/ensure_dir.ts';
+import { copy } from 'https://deno.land/std@0.136.0/streams/conversion.ts';
+
+export const exec = async (opts: { cmd: string, cwd?: string }) => {
+  const runner =  Deno.run({
```

Comment:
This API was deprecated, wasn't it?

Reply:
https://doc.deno.land/deno/stable/~/Deno.run

Nope.

Reply:
Right, let's leave it this way for now. The API is going to replaced soon, though.

https://deno.com/blog/v1.21#new-unstable-api-for-subprocesses